modules/seqexec/server/src/main/scala/seqexec/server/keywords/ObsKeywordReader.scala

```diff
@@ -295,10 +295,7 @@ object ObsKeywordReader extends ObsKeywordsReaderConstants {
 
     override def headerPrivacy: F[Boolean] = headerPrivacyF
 
-    private val noProprietaryMonths: F[String] =
-      LocalDate.now(ZoneId.of("GMT")).format(DateTimeFormatter.ISO_LOCAL_DATE).pure[F]
-
-    private val calcProprietaryMonths: F[String] =
+    private val calcReleaseDate: F[String] =
       EitherT(
         config.extractAs[Integer](PROPRIETARY_MONTHS_KEY)
           .recoverWith {
```

Comment:
There is a call to LocalDate.now inside.We should put that inside a `Sync.delay`